### PR TITLE
1345 ts test type defination upgrade

### PR DESCRIPTION
**Pull request by wli-pro**
follow-up PR for #1345 after this PR: https://github.com/litentry/litentry-parachain/pull/1643

Added prettier for generate script so they are consistent.

See commits history for their effect.

### CODE REVIEW

File: tee-worker/ts-tests/package.json [hunk 1]
```diff
@@ -20,8 +21,9 @@
 	},
 	"dependencies": {
 		"@noble/ed25519": "^1.7.3",
-		"@polkadot/api": "^9.14.2",
-		"@polkadot/types": "^9.14.2",
+		"@polkadot/api": "^10.3.4",
```
Comment (0xverin) [inline tee-worker/ts-tests/package.json:24]: Can ^10.x version work now?

Comment (Kailai-Wang) [reply tee-worker/ts-tests/package.json:24]: @Verin1005 https://github.com/litentry/litentry-parachain/pull/1643#issuecomment-1530368434

File: tee-worker/ts-tests/package.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
 	"license": "ISC",
 	"scripts": {
 		"load-meta": "curl -s -H \"Content-Type: application/json\" -d '{\"id\":\"1\", \"jsonrpc\":\"2.0\", \"method\": \"state_getMetadata\", \"params\":[]}' http://localhost:9944 > litentry-metadata.json",
-		"generate": "yarn generate:defs && yarn generate:meta",
+		"format": "yarn prettier --write .",
+		"generate": "yarn generate:defs && yarn generate:meta && yarn prettier --write .",
```
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/package.json]: We can't say `&& yarn format` here? :thinking:

**Review by arieljbirnbaum — APPROVED** [review]
:raised_hands:

File: tee-worker/ts-tests/package.json (hunk 2 above)
Comment (wli-pro) [reply tee-worker/ts-tests/package.json]: well spot, updated now.

**Pull request merged**